CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+Changes by Version
+==================
+
+v0.31.0 (2018-01-12)
+--------------------
```

Comment:
Could we add one list item per added change? All the following would be in the one list item. Also, add a link to the PR, in some projects there is also a name of the commiter.

Two major changes are missing there:
* introduction of log fields
* removed `span.log(object)` methods

Reply:
I'm not sure what you mean by "All the following would be in the one list item." Do you mean only one list item for all the Scpoce/ActiveSpan changes?

Reply:
Yes as it was part of one PR

Reply:
I looked though the PRs... unfortunately I don't think the Pull Requests line up cleanly, as we went through many revisions. I will try to find a middle ground.